pkg/logs/config/integration_config_test.go

```diff
@@ -103,6 +103,10 @@ func TestBuildLogsAgentIntegrationConfigsWithMisconfiguredFile(t *testing.T) {
 	ddconfdPath = filepath.Join(testsPath, "misconfigured_5", "conf.d")
 	_, err = buildLogSources(ddconfdPath, false)
 	assert.NotNil(t, err)
+
+	ddconfdPath = filepath.Join(testsPath, "misconfigured_6")
+	_, err = buildLogSources(ddconfdPath, false)
+	assert.NotNil(t, err)
```

Comment:
While this does test the behavior, I think it'd be better to do a unit test - test `validateProcessingRules` only. It'll be more concise, easier to update